src/benchmark.cc

```diff
@@ -906,6 +895,13 @@ void ParseCommandLineFlags(int* argc, const char** argv) {
   }
 }
 
+Benchmark* RegisterBenchmarkInternal(Benchmark* bench) {
```

Comment:
to document the ownership passing, this could take a `std::unique_ptr<Benchmark>` directly.